src/main/java/org/broadinstitute/hellbender/tools/HaplotypeCallerSpark.java

```diff
@@ -136,13 +134,6 @@ public boolean requiresReference(){
 
     @Override
     protected void runTool(final JavaSparkContext ctx) {
-        //TODO remove me when https://github.com/broadinstitute/gatk/issues/4274 and https://github.com/broadinstitute/gatk/issues/4303 are fixed
```

Comment:
I think we should keep the upfront warning here.  It's much better to fail instantly instead of at the end when we try to write the file.

Reply:
Agreed. Fixed.